lib/request.js

```diff
@@ -0,0 +1,8 @@
+import { request } from "@octokit/request";
+
+export default request.defaults({
+  baseUrl: process.env["GITHUB_API_URL"],
+  headers: {
+    "user-agent": "actions/create-github-app-token",
```

Comment:
we can add the current version here with https://github.com/gr2m/semantic-release-plugin-update-version-in-files, but I'd do that in a separate pr.